src/lsptoolshost/roslynLanguageServer.ts

```diff
@@ -672,6 +685,30 @@ export async function activateRoslynLanguageServer(
 
     // Start the language server.
     _languageServer.start();
+
+    return _languageServer;
+
+    function scanExtensionPlugins(): string[] {
+        return vscode.extensions.all.flatMap((extension) => {
+            const loadPaths = extension.packageJSON.contributes?.['csharpExtensionLoadPaths'];
+            if (loadPaths === undefined || loadPaths === null) {
+                _traceChannel.appendLine(`Extension ${extension.id} does not contribute csharpExtensionLoadPaths`);
```

Comment:
Given this is the common case, just skip the logging here?

Reply:
I figure that's what trace logging is for: the common case that someone is trying to debug what's happening.

Reply:
Is the _traceChannel though the LSP channel? @dibarbet?